packages/headless/src/features/history/history-actions-loader.ts

```diff
@@ -0,0 +1,40 @@
+import {AsyncThunkAction} from '@reduxjs/toolkit';
+import {Engine} from '../../app/headless-engine';
+import {history} from '../../app/reducers';
+import {back, forward} from './history-actions';
+
+/**
+ * The history action creators.
+ */
+export interface HistoryActionCreators {
+  /**
+   * Moves backward in the interface history.
```

Comment:
is interface really a good word here inside headless, we could remove it

Reply:
ok, will keep as-is and we can revise in the future,